spec/services/atmosphere/flavor_manager_spec.rb

```diff
@@ -3,81 +3,81 @@
+    context 'when populating a tenant with flavors' do
+      let!(:flavor6_ost) { create(:virtual_machine_flavor, tenant: t1, id_at_site: "6", flavor_name: "foo") }
+      let!(:flavor6_aws) { create(:virtual_machine_flavor, tenant: t2, id_at_site: "m1.medium", flavor_name: "foo") }
 
       before do
-        create(:virtual_machine_flavor, compute_site: cs1, id_at_site: "5", flavor_name: "bar")
+        create(:virtual_machine_flavor, tenant: t1, id_at_site: "5", flavor_name: "bar")
       end
 
-      it 'registers compute sites with flavors' do
-        expect(cs1.virtual_machine_flavors.count).to eq 2
-        Atmosphere::FlavorManager::scan_site(cs1)
-        expect(cs1.virtual_machine_flavors.count).to eq cs1.cloud_client.flavors.count
+      it 'registers tenants with flavors' do
+        expect(t1.virtual_machine_flavors.count).to eq 2
+        Atmosphere::FlavorManager::scan_tenant(t1)
+        expect(t1.virtual_machine_flavors.count).to eq t1.cloud_client.flavors.count
       end
 
       it 'updates existing flavour on openstack' do
-        Atmosphere::FlavorManager::scan_site(cs1)
-        fog_flavor = cs1.cloud_client.flavors.detect { |f| f.id == flavor6_ost.id_at_site }
+        Atmosphere::FlavorManager::scan_tenant(t1)
+        fog_flavor = t1.cloud_client.flavors.detect { |f| f.id == flavor6_ost.id_at_site }
 
         flavor6_ost.reload
         expect(flavor6_ost).to flavor_eq fog_flavor
       end
 
       it 'updates existing flavour on aws' do
-        Atmosphere::FlavorManager::scan_site(cs2)
-        fog_flavor = cs2.cloud_client.flavors.detect { |f| f.id == flavor6_aws.id_at_site }
+        Atmosphere::FlavorManager::scan_tenant(t2)
+        fog_flavor = t2.cloud_client.flavors.detect { |f| f.id == flavor6_aws.id_at_site }
 
         flavor6_aws.reload
         expect(flavor6_aws).to flavor_eq fog_flavor
       end
 
       it 'removes nonexistent flavor on openstack' do
-        Atmosphere::FlavorManager::scan_site(cs1)
-        expect(cs1.virtual_machine_flavors.count).to eq cs1.cloud_client.flavors.count
-        # Add nonexistent flavor to compute_site cs1
-        create(:virtual_machine_flavor, compute_site: cs1, id_at_site: "foo", flavor_name: "baz")
-        cs1.reload
-        expect(cs1.virtual_machine_flavors.where(flavor_name: "baz").count).to eq 1
-        # Scan site again and expect flavor "baz" to be gone
-        Atmosphere::FlavorManager::scan_site(cs1)
-        cs1.reload
-        expect(cs1.virtual_machine_flavors.where(flavor_name: "baz").count).to eq 0
+        Atmosphere::FlavorManager::scan_tenant(t1)
+        expect(t1.virtual_machine_flavors.count).to eq t1.cloud_client.flavors.count
+        # Add nonexistent flavor to tenant t1
+        create(:virtual_machine_flavor, tenant: t1, id_at_site: "foo", flavor_name: "baz")
+        t1.reload
+        expect(t1.virtual_machine_flavors.where(flavor_name: "baz").count).to eq 1
```

Comment:
Line is too long. [82/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.